src/Tribe/Shortcodes/User_Event_Confirmation_List.php

```diff
@@ -16,24 +16,22 @@
  *     [tribe-user-event-confirmations limit="16"]
  */
 class Tribe__Tickets__Shortcodes__User_Event_Confirmation_List {
+	protected $shortcode_name = 'tribe-user-event-confirmations';
 	protected $params = array();
 
 	/**
-	 * Registers a user event confirmation list shortcode using the specified
-	 * name (or else defaults to "tribe-user-event-confirmations").
-	 *
-	 * @param string $shortcode_name
```

Comment:
thanks @bordoni I was not aware of the use of @since will keep it in mind for the future. Can you re-check if I understood this correctly?